PhysicsTools/NanoAOD/python/protons_cff.py

```diff
@@ -0,0 +1,73 @@
+import FWCore.ParameterSet.Config as cms
+from PhysicsTools.NanoAOD.common_cff import *
+from Configuration.Eras.Modifier_run2_miniAOD_80XLegacy_cff import run2_miniAOD_80XLegacy
+from Configuration.Eras.Modifier_run2_nanoAOD_94XMiniAODv1_cff import run2_nanoAOD_94XMiniAODv1
+from Configuration.Eras.Modifier_run2_nanoAOD_94XMiniAODv2_cff import run2_nanoAOD_94XMiniAODv2
+from Configuration.Eras.Modifier_run2_nanoAOD_94X2016_cff import run2_nanoAOD_94X2016
```

Comment:
@jwill24 
please define the run2_nanoAOD_102Xv1 here

Reply:
@mariadalfonso Yes, sorry, I forgot to import it. 
This has been corrected.